addons/account/models/account_move.py

```diff
@@ -104,10 +104,6 @@ def _get_default_journal(self):
 
         return journal
 
-    @api.model
-    def _get_default_invoice_date(self):
-        return fields.Date.context_today(self) if self._context.get('default_move_type', 'entry') in self.get_purchase_types(include_receipts=True) else False
```

Comment:
you can't remove a method in a stable version: Add a `# TO BE REMOVED IN MASTER` instead and remove it during the fw-port